web/src/store/GlobalStore.js

```diff
@@ -58,6 +58,8 @@ function reducer(state, action) {
       }
     }
     case actions.LOGOUT:
+      window.localStorage.removeItem("projects")
+      window.localStorage.removeItem("uiFilters")
```

Comment:
If you have this 2 items only in the localStorage, you can use `window.localStorage.clear()`

Reply:
> If you have this 2 items only in the localStorage, you can use `window.localStorage.clear()` (it remove all your items in the localStorage)

@clegirar 
No because there is 3 keys (`theme`, `projects` and `uiFilters`) and I don't want to clear `theme`.